src/Tests/Microsoft.Diagnostics.Monitoring.UnitTests/AuthenticationTests.cs

```diff
@@ -185,6 +185,30 @@ public async Task ApiKeyAuthenticationSchemeTest()
             Assert.NotNull(processes);
         }
 
+        /// <summary>
+        /// Tests that API key authentication can be configured correctly and
+        /// that the key can be rotated wihtout shutting down dotnet-monitor.
+        /// </summary>
+        [Fact]
+        public async Task TempApiKeyAuthenticationSchemeTest()
```

Comment:
nit: TempApiKeyTest, since TempApiKey is not a scheme itself. Also, please update the test xml docs to reflect what the test is doing.